src/main/java/com/github/manolo8/darkbot/backpage/BackpageManager.java

```diff
@@ -124,6 +128,23 @@ public HttpURLConnection getConnection(String params) throws Exception {
         return conn;
     }
 
+    public HttpURLConnection getGalaxyConnection(String params, int minWait) throws Exception {
+        Time.sleep(lastGalaxyRequest + minWait - System.currentTimeMillis());
+        return getGalaxyConnection(params);
+    }
+
+    public HttpURLConnection getGalaxyConnection(String params) throws Exception {
+        if (isInvalid()) throw new UnsupportedOperationException("Can't connect when sid is invalid");
+        HttpURLConnection conn = (HttpURLConnection) new URL(this.instance + params).openConnection();
+
+        conn.setInstanceFollowRedirects(false);
+        conn.setRequestProperty("Cookie", "dosid=" + this.sid);
+        conn.setRequestProperty("User-Agent", "Mozilla/5.0");
```

Comment:
just make the galaxyManager wrap getConnection()  and add the request params itself instead of having a different method and counter for it